lib/src/components/NxSidebarNavigation/NxSidebarNavigation.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { faBars, faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+
+import NxButton from '../NxButton/NxButton';
+import NxCloseButton from '../NxCloseButton/NxCloseButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, SidebarNavLinkProps, propTypes } from './types';
+
+import './NxSidebarNavitation.scss';
+
+const NxSidebarNavLink = function NxSidebarNavLink({ name, href, icon, current }: SidebarNavLinkProps) {
+  const navigationLinkClassnames = classnames('nx-sidebar-nav--item', {
+    'selected': current
+  });
+
+  return (
+    <li className={navigationLinkClassnames} key={`nav-link-${name}`}>
+      <a href={href} className="nx-sidebar-nav--link">
+        <NxFontAwesomeIcon icon={icon} />
+        <span className="nx-sidebar-nav--link-name">{name}</span>
+      </a>
+    </li>
+  );
+};
+
+const NxSidebarNavigation = function NxSidebarNavigation(props: Props) {
+  const {
+    isOpen,
+    className,
+    onToggleClick,
+    logoImg,
+    logoText,
+    logoLink,
+    links,
+    helpLink,
+    helpText,
+    collapsedReleaseText,
+    expandedReleaseText,
+    attributions
+  } = props;
+
+  const classes = classnames(className, 'nx-sidebar-nav', {
+    'nx-sidebar-nav__open': isOpen,
+    'nx-sidebar-nav__closed': !isOpen
```

Comment:
These look like they are supposed to be Modifiers, not Elements

Reply:
> These look like they are supposed to be Modifiers, not Elements

Changed in 232611c992543e74499f2aab8381c27efdf6e960